client/pdr.py

```diff
@@ -0,0 +1,64 @@
+'''
+By S.Rees 2023 Python PDR Scraper.
+
+For use on pager decoder devices running from a serial device
+
+Edit as required'''
+
+import sys
+import os
+sys.path.append(os.path.realpath('.'))
+import serial
+import time
+Import requests
+from datetime import datetime
+import re
+import inquirer
+from serial.tools import list_ports
+questions = [
+    inquirer.List('device',
+                  message="Please pick a device",
+                  choices=[port.device+ ":"+ port.description 
+                  for port in list_ports.comports()],
+              ),
+]
+answers = inquirer.prompt(questions)
+dev = answers["device"].split(":", 1)
+ser = serial.Serial(dev[0], 19200, timeout=5)
+from clint.textui import puts, colored
+
+def post(flexcode, msg, when):
+		headers = {
+			'X-Requested-With': 'XMLHttpRequest',
+			'apikey': "", #pagermon APIKey
+			'User-Agent': 'PagerMon pdr.py',
+		}
+		params = {
+			"address": flexcode,
+			"message": msg,
+			"datetime": when,
+			"source": "",
+			}
+		requests.post('http://Your IP Address Here/api/messages', data=params, headers=headers)
```

Comment:
Suggest pulling the following data from /client/config/default.json so its consistent with the rest of the project

* IP Addresss/Hostname
* API Key
* Source/identifier

Reply:
> Suggest pulling the following data from /client/config/default.json so its consistent with the rest of the project
> 
> * IP Addresss/Hostname
> * API Key
> * Source/identifier

When testing this was the json results pulled. When you test you will see these are the required arguments.

Reply:
What i'm trying to suggest is to parse the default.json and config.json file to get the Hostname, API Key & Source/Identifier instead of having to put it in pdr.py manually..

Reply:
> What i'm trying to suggest is to parse the default.json and config.json file to get the Hostname, API Key & Source/Identifier from the config and use variables in pdr.py for each one instead of having to put it in pdr.py manually..

Happy to do so. I'll add it as an update (got a bit on)